src/BlockEntities/DispenserEntity.cpp

```diff
@@ -211,7 +211,9 @@ void cDispenserEntity::DropSpenseFromSlot(cChunk & a_Chunk, int a_SlotNum)
 				DropFromSlot(a_Chunk, a_SlotNum);
 				break;
 			}
-			if (m_World->GrowRipePlant(dispAbsCoord.x, dispAbsCoord.y, dispAbsCoord.z, true))
+
+			// Simulate a right-click with bonemeal:
+			if (m_World->GrowRipePlant(dispAbsCoord))
```

Comment:
Does this actually create the intended behaviour

Reply:
Fixed thanks